mlrun/api/schemas/feature_store.py

```diff
@@ -56,3 +88,20 @@ class Config:
 
 class FeatureSetsOutput(BaseModel):
     feature_sets: List[FeatureSet]
+
+
+class FeatureSetDigestOutput(BaseModel):
+    name: str
+    tag: Optional[str]
+    labels: Optional[dict]
+    uid: Optional[str]
+    entities: List[Feature]
```

Comment:
Let's reuse already existing schemas if possible, I suggest to simply use the regular `metadata: FeatureSetMetadata` here, plus I think I would make it `spec: FeatureSetDigestSpec`

Reply:
Per our offline discussion - will make the changes.